Developer Manual

benbjohnson opened the pull request.

jhelmsen commented:
Great work Ben! Excellent!  This captures all of the technical essence of how to write operations for the system.  I would recommend that we also call out particular formats, and where to find them, in this document as examples for interested parties to follow.  I know that it would already be in the user guide, but this is a good place to repeat that for interested parties. (See the latest message from Psiphon where they refer to this pull request.)

Finally, can we include a section on how to include new plugins or add routines to existing plugins by  expanding the plugin registry?

benbjohnson commented:
> _I would recommend that we also call out particular formats_

I added a _References_ section that gives some idea of where to find different functions in practice.

> Can we include a section on how to include new plugins or add routines to existing plugins by expanding the plugin registry?

Yes, I added a _ Plugin Development_ section.
